docs/msi_v2/slc_revocation_spec.md

```diff
@@ -1,46 +1,46 @@
-# Short-Lived Credential (SLC) Revocation Specification
+# Certificate Revocation Specification
```

Comment:
The name of the file should also be changed.

Reply:
updated